.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Got some code for us? Awesome 🎊!
+
+Please include a description of your change & check your PR against this list, thanks!
```

Comment:
I often find it hard to understand what _problem_ is being solved with PR’s. Perhaps we can ask to provide that?